packages/utils/schema/user-profile.graphql

```diff
@@ -0,0 +1,111 @@
+type ProfileName
+  @createModel(accountRelation: SINGLE, description: "Your name") {
+  name: String! @string(maxLength: 150)
+}
+
+type ProfileAvatar
+  @createModel(
+    accountRelation: SINGLE
+    description: "An image and optional fallback images representing your avatar"
+  ) {
+  original: ImageMetadata!
+  alternatives: [ImageMetadata] @list(maxLength: 3)
+}
+
+type ProfileDescription
+  @createModel(
+    accountRelation: SINGLE
+    description: "A short profile bio / blurb / description"
+  ) {
+  description: String! @string(maxLength: 420)
```

Comment:
In general, I'd like to err on the side of extravagance when imposing size restrictions.

It's important to remember that we're dealing with text characters. The amount of space they take up is trivial compared to video or another heavyweight.

```suggestion
  description: String! @string(maxLength: 6942069)
```